### ISSUE TITLE
Too Much File Preview Spacing

### ISSUE THREAD

**Issue by Bamboy** [gitahead/gitahead#347]
Linux v2.6.0

There is an awful lot of wasted space here between the files, wouldn't you agree?

![Untitled](https://user-images.githubusercontent.com/3387750/71302216-6e703700-235d-11ea-9594-0c1082baa581.png)

**Comment by JakeSays** [comment]
I agree wholeheartedly with this. All that space and then having to manually collapse the diffs makes this view completely useless. Please add a view that just displays the changed files.

(There may be a way to collapse all the diffs at once, but I haven't found it)

**Comment by Murmele** [comment]
> I agree wholeheartedly with this. All that space and then having to manually collapse the diffs makes this view completely useless. Please add a view that just displays the changed files.
> 
> (There may be a way to collapse all the diffs at once, but I haven't found it)

The listwidget above the diff. Shows only the changed files. You can click on it to see the diffs of only one file
